core/cap-0067.md

```diff
@@ -213,31 +228,40 @@ The trades within a path payment are conceptually between the source account and
 
 Note that if the path payment has an empty path and `sendAsset == destAsset`, then the operation is effectively a regular [payment](#payment), so emit an event following the specifications of the payment section.
 
+As specified in the [section to update the transfer events data field](#Update-the-data-field-in-the-transfer-event-from-an-integer-to-a-map-to-store-additional-memo-information), both `from_muxed_id` and `to_muxed_id` can be added to the `data` map on the `transfer` event. More specifically, if the source account is muxed and appears as `from` or `to` in an event, the respective `from_muxed_id` or `to_muxed_id` will be set. For the final event to the destination, if it is a `transfer`, `to_muxed_account` will be set if the `destination` is muxed, and if not, then it'll be set if the tx memo is set.
 
 #### Create Account
 Emit the following event:
 ```
-contract: native asset, topics: ["transfer", from:Address, to:Address, sep0011_asset:String], data: amount:i128
+contract: native asset, topics: ["transfer", from:Address, to:Address, sep0011_asset:String], data: { amount:i128 }
 ```
 
 * `from` is the account being debited (creator).
 * `to` is the account being credited (created).
 * `amount` is the starting native balance.
 
+Note that no event will be emitted if the `amount` is 0.
```

Comment:
Should we make this true in general? Seems a bit odd to specialize this edge case only here.

Reply:
Hmm yeah you bring up a good point. The SAC `transfer` function always emits the event, even with a 0 amount. I believe this should also be specified in SEP-0041 (I'll check). So for consistency, I'll remove this.